.github/workflows/snap.yaml

```diff
@@ -0,0 +1,49 @@
+name: snap
+
+on:
+  # Manually trigger this from the Actions tab
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release Type'
+        required: true
+        default: 'stable'
+        type: choice
+        options:
+        - lts
+        - stable
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        release:
+        - ${{ inputs.release }}
+        platform:
+        - i386
+        - amd64
+        - armhf
+        - arm64
+        exclude:
+        - release: stable
+          platform: armhf
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v3
+    - name: Set up snapcraft.yaml
+      run: |
+        cp "snap/snapcraft-${{ inputs.release }}.yaml" "snap/snapcraft.yaml"
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Build Snap
+      uses: diddlesnaps/snapcraft-multiarch-action@v1
+      id: snapcraft
+      with:
+        architecture: ${{ matrix.platform }}
+    - name: Upload and Release
+      uses: snapcore/action-publish@v1
+      with:
+        store_login: ${{ secrets.STORE_TOKEN }}
+        snap: ${{ steps.snapcraft.outputs.snap }}
+        release: ${{ inputs.release == 'lts' && 'lts/stable' || inputs.release }}
```

Comment:
This will return `lts/stable` if we trigger the `lts` workflow or `stable` (I could have used that instead of `inputs.release`). This is the channel to which you want to upload the snap, by default if you only specify the *risk level* as `stable`, `candidate`, `beta` or `edge` it will upload to `latest/stable, `latest/candidate`, `latest/beta` or `latest/edge` respectively. If you want to use a different track you have to specify both the track and the risk level.

Reply:
Ah, okay. I guess I'm too used to parsing `&&` and `||` like in Julia 😄